.github/workflows/ci.yml

```diff
@@ -13,159 +10,78 @@ on:
       - main
 
 jobs:
-  pnpmstore:
+  # "checks" job runs on linux + node14 only and checks that install, build, lint and audit work
+  # it also primes the pnpm store cache for linux, important for downstream tests
+  checks:
     timeout-minutes: 5
     runs-on: ${{ matrix.os }}
-
     strategy:
-      fail-fast: false
       matrix:
-        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        os: [ ubuntu-latest ]
         node: [ 14 ]
```

Comment:
```suggestion
        node: [ 14 ]
        os: [ ubuntu-latest ]
```

Flipping this would make github action's output a bit consistent :grin: 

![image](https://user-images.githubusercontent.com/34116392/128480500-9ecc6c38-c48d-4360-9f6f-8836832528da.png)

Reply:
done